functions/pipeline/onboarding/__init__.py

```diff
@@ -9,10 +26,106 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         req_body = req.get_json()
         logging.error(req.get_json())
         url_list = req_body["imageUrls"]
-        url_string = (", ".join(url_list))
     except ValueError:
         print("Unable to decode JSON body")
         return func.HttpResponse("Unable to decode POST body", status_code=400)
 
     logging.error(req_body)
-    return func.HttpResponse("Got body: " + url_string, status_code=200)
\ No newline at end of file
+
+    # Build list of image objects to pass to DAL for insertion into DB.
+    image_object_list = []
+    image_name_list = []
+    
+    # TODO: Add check to ensure image URLs sent by client are all unique.
+
+    for url in url_list:
+        # Split original image name from URL
+        original_filename = url.split("/")[-1]
+        image_name_list.append(original_filename)
+        # Create ImageInfo object (def in db_access.py)
+        # Note: For testing, default image height/width are set to 50x50
+        # TODO: Figure out where actual height/width need to come from
+        image = DB_Access_V2.ImageInfo(original_filename, url, 50, 50)
+        # Append image object to the list
+        image_object_list.append(image)
+
+    logging.info("Now connecting to database...")

```

Comment:
@ackaufman the db access portion should also be in a try/catch and send the appropriate http response if if when an error

Reply:
TODO note added